apps/condo/domains/acquiring/schema/Payment.test.js

```diff
@@ -209,6 +209,19 @@ describe('Payment', () => {
                 expect(updatedPayment).toBeDefined()
                 expect(updatedPayment).toEqual(expect.objectContaining(payload))
             })
+            test('admin can\'t update organization field', async () => {
```

Comment:
Could you please also write test for `disconnect` case

Reply:
:thum